src/PrestaShopBundle/Entity/Lang.php

```diff
@@ -300,6 +301,14 @@ public function getIsRtl()
         return $this->isRtl;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function isRTL()
+    {
+        return $this->getIsRtl();
```

Comment:
```diff
- return $this->getIsRtl();
+ return (bool) $this->getIsRtl();
```

Reply:
In antoher piece of code you wanted me to get reed of the `(bool)` cast, why add it here?